### ISSUE TITLE
Comments polluting the source code of the example

### ISSUE THREAD

**Issue by spinningarrow** [sourcejs/sourcejs-comments#17]
The comment markers are added a lot of unnecessary code to the example, which can be seen when the _Show source code_ option in the sidebar is turned on. The markup looks like this:

```
<div class="source-bbl_w" timestamp="1449639068335" id="5667bca0ae41c23f4d6f6f49" style="left: 334px; top: 35px;">
  <div class="js-bbl">
    <div class="source-bbl_form" style="display: none;">
      <textarea class="source-bbl_it" placeholder="Your comment..."></textarea>
      <div class="source-bbl_sep"></div>
      <input class="source-bbl_name" placeholder="Signature"/>
      <div class="source-bbl_actions">
        <button class="js-bbl_submit">Save</button>
        <a href="#" class="js-bbl_cancel">Cancel</a>
      </div>
    </div>
    <div class="source-bbl_info __show">
      <div class="source-bbl_tx">what do you think?</div>
      <div class="source-bbl_author">Sahil</div>
      <div class="js-bbl_close"></div>
    </div>
  </div>
  <a href="#" class="js-bbl_point"></a>
</div>
```

Can I hide that somehow?

**Comment by operatino** [comment]
Hm, thats' a good point, I'll take a look how we can move it from example block.

**Comment by operatino** [comment]
Fixed in `0.2.0` release, now comments are placed outside `source_example` block.
